src/boosteriferous/boosteriferous.csproj

```diff
@@ -27,14 +27,34 @@
     <ConsolePause>false</ConsolePause>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
     <Reference Include="Assembly-CSharp">
-      <HintPath>..\lib\Assembly-CSharp.dll</HintPath>
-      <Private>False</Private>
+      <HintPath>R:\KSP_1.1.3_dev\KSP_x64_Data\Managed\Assembly-CSharp.dll</HintPath>
```

Comment:
Your personal reference paths aren't likely to be useful for anyone else, especially when they're absolute paths starting with a drive letter.  (*Real* filesystems have a global root, called `/`)  That's kinda why I did the src/lib/ thing in the first place, that way there's at least a well-defined place for them to be.